stl/inc/format

```diff
@@ -978,22 +978,45 @@ _NODISCARD constexpr bool _Is_execution_charset_self_synchronizing() {
 #endif // ^^^ EDG workaround ^^^
 }
 
-inline constexpr char32_t _Width_estimate_intervals[] = { // Per N4928 [format.string.std]/12
-    0x1100u, 0x1160u, 0x2329u, 0x232Bu, 0x2E80u, 0x303Fu, 0x3040u, 0xA4D0u, 0xAC00u, 0xD7A4u, 0xF900u, 0xFB00u, 0xFE10u,
-    0xFE1Au, 0xFE30u, 0xFE70u, 0xFF00u, 0xFF61u, 0xFFE0u, 0xFFE7u, 0x1F300u, 0x1F650u, 0x1F900u, 0x1FA00u, 0x20000u,
-    0x2FFFEu, 0x30000u, 0x3FFFEu};
+// Generated per N4950 [format.string.std]/13, by tools/unicode_properties_parse/format_width_estimate_intervals.cpp

```

Comment:
Should we also mention that the data is based on UCD v15.0.0 here?

Reply:
We don't think that's necessary, and it would lead to yet another place that needs to be updated when regenerating the Unicode data for a new version. Mentioning v15 in the generator directory is sufficient.